src/cc/api/BPFTable.h

```diff
@@ -48,7 +48,7 @@ class BPFQueueStackTableBase {
   StatusTuple leaf_to_string(const ValueType* value, std::string& value_str) {
     char buf[8 * desc.leaf_size];
     StatusTuple rc = desc.leaf_snprintf(buf, sizeof(buf), value);
-    if (!rc.code())
+    if (!rc.ok())
```

Comment:
`!rc.code()` is equivalent to `rc.code() == 0`. So this should be `rc.ok()`.

Reply:
Oh, you are right. I must have went through a weird mental wiring that exactly reversed the meaning of 0...